lutron-aurora/profiles/lutron-aurora-profile.yml

```diff
@@ -0,0 +1,30 @@
+name: lutron-aurora.v1
+components:
+- id: main
+  capabilities:
+    - id: button
+      version: 1
+    - id: switchLevel
```

Comment:
did you also want to emulate "switch"?

Reply:
It is sort of up in the air, this is currently setup to just be a button and a dimmer but it _could_ be a switch instead